splash/tests/test_execute.py

```diff
@@ -559,22 +559,14 @@ def test_dateobj(self):
 
     def test_regexp(self):
         self.assertEvaljsResult(
-            '/my-regexp/i',
-            {
-                u'_jstype': u'RegExp',
-                'caseSensitive': False,
-                'pattern': u'my-regexp'
-            },
+            'var a = "123A"; a.match(/[a]/i);',
+            ['A'],
```

Comment:
This is a backwards-incompatible change; it requires changes to docs. It is fine to remove this feature (it is not that useful), but could you please check why does it work differently with pyqt5/qt5?